core/src/main/java/org/apache/brooklyn/entity/group/DynamicCluster.java

```diff
@@ -148,6 +148,11 @@
     ConfigKey<EntityFactory> FACTORY = ConfigKeys.newConfigKey(
             EntityFactory.class, "dynamiccluster.factory", "factory for creating new cluster members", null);
 
+
+    @SetFromFlag("removalStrategy")
```

Comment:
`@SetFromFlag("removalStrategy")` is on both this and the deprecated key. Which one wins?